packages.gradle

```diff
@@ -394,6 +416,8 @@ def genTasks = { target ->
     delete ("$PKG_BUILD_DIR/deb")
     def final DEB_BLD_DIR = "$PKG_BUILD_DIR/deb/$NAME-${PKG_VERSION}"
     def final DEB_PKG_DIR = "$PKG_BUILD_DIR/deb/$PKG_NAME-${PKG_VERSION}-${BIGTOP_BUILD_STAMP}"
+    def final ENABLE_MAVEN_PARALLEL_BUILD = config.bigtop.components[target].maven_parallel_build
+    def final MAVEN_BUILD_THREADS = project.hasProperty('buildThreads') ? project.property('buildThreads') : null
```

Comment:
Documentation/comments for the feature could be added later. We do not allow number without `C` while [Maven do](https://cwiki.apache.org/confluence/display/MAVEN/Parallel+builds+in+Maven+3)?